src/js/components/modals/AlertModalComponent.jsx

```diff
@@ -0,0 +1,54 @@
+var React = require("react/addons");
+var Util = require("../../helpers/Util");
+
+var ModalComponent = require("../ModalComponent");
+
+var AlertModalComponent = React.createClass({
+  displayName: "AlertModalComponent",
+
+  propTypes: {
+    message: React.PropTypes.string,
+    onDestroy: React.PropTypes.func
+  },
+
+  componentDidMount: function () {
+    this.refs.button.getDOMNode().focus();
```

Comment:
Let's use `React.findDOMNode`: https://facebook.github.io/react/docs/working-with-the-browser.html#refs-and-finddomnode

https://github.com/mesosphere/marathon-ui/pull/175